api/api.go

```diff
@@ -43,6 +43,7 @@ const (
 	headerKeyContentLength                = "Content-Length"
 	defaultVolumesPath                    = "/ifs/volumes"
 	defaultVolumesPathPermissions         = "0777"
+	defaultUnresolvableHosts              = false
```

Comment:
can you also rename this var to defaultIgnoreUnresolvableHosts? Similar comment for other places also